advanced.md

```diff
@@ -167,4 +167,112 @@ public void toggleAllDigitalPins( bool setPinsHigh )
 		firmata.sendDigitalPort( 1, 0 );	//all pins low
```

Comment:
Since the method parameter is uint8_t, it would require:
static_cast<uint8_t>( Firmata::PinState::LOW )
and I think it misrepresents the purpose of the State enums. There are 256 possible combinations of values to use here, and using LOW only works for one.